.gitignore

```diff
@@ -170,6 +170,7 @@ dmypy.json
 
 secrets/
 data/mysql
```

Comment:
since we are migrating from mysql to postgres, not sure if we really need this here. There is no harm in keeping it.

Reply:
Good point, we can scrap this